src/main/java/core/basesyntax/MainApp.java

```diff
@@ -12,5 +12,15 @@
  * - Отправить все машини на работу и остановить работу всех машин.
  */
 public class MainApp {
-
+    public static void main(String[] args) {
```

Comment:
add all classes in PR (Excavator, Bulldozer, Track, Machine)

Reply:
done